static_src/test/unit/stores/space_store.spec.js

```diff
@@ -73,6 +75,33 @@ describe('SpaceStore', function() {
     });
   });
 
+  describe('on fetch all for orgs', function() {
+    it('should call space fetch for each space that belongs to the org', () => {
+      const orgGuid = '240jejbvrl';
+      SpaceStore.push({ guid: 'abc1', organization_guid: orgGuid });
+      SpaceStore.push({ guid: 'abc2', organization_guid: orgGuid });
+      SpaceStore.push({ guid: 'abc3', organization_guid: '23fdg' });
+
+      const spy = sandbox.stub(cfApi, 'fetchSpace').returns(Promise.resolve());
+
+      spaceActions.fetchAllForOrg(orgGuid);
+
+      expect(spy).toHaveBeenCalledTwice();
+    });
+
+    it('should emit change if there are spaces to be fetched', () => {
+      const orgGuid = '240jejxcb';
+      SpaceStore.push({ guid: 'abc0', organization_guid: orgGuid });
+
+      const spy = sandbox.spy(SpaceStore, 'emitChange');
+
+      spaceActions.fetchAllForOrg('sdf');
+      spaceActions.fetchAllForOrg(orgGuid);
+
+      expect(spy).toHaveBeenCalledOnce();
+    });
```

Comment:
I don't see an `emitChange` call in the `SPACES_FOR_ORG_FETCH` handler, is this event from the `load` call? Maybe we should update the test description or add a comment so the test is more clear.